conjure-test/build.rs

```diff
@@ -7,7 +7,6 @@ fn main() {
 
     println!("cargo:rerun-if-changed={}", input);
     conjure_codegen::Config::new()
-        .run_rustfmt(false)
```

Comment:
Out of curiosity, why don't we need this anymore?

Reply:
Disabling rustfmt makes the test build a tiny bit slower, but it makes compilation errors totally unreadable when you're messing with the codegen since it's all on one huge line before reformatting.

Reply:
a tiny bit faster*